io/io-q&a.md

```diff
@@ -114,7 +114,7 @@
 
 1. Ce întoarce o operație asincronă?
 
-   Operațiile asincrone întorc o promisiune, faptul că la un moment dat, operația va fi satisfăcută.
+   Operațiile asincrone întorc un cod de eroare, 0 sau -1.
```

Comment:
Corect, nu intorc date deloc, doar aceste 2 coduri. Datele pot fi accesate printr-un eventfd.

Reply:
@valy3124 Exactly my point. Add this info to your commit, otherwise the explanation is too terse and counterintuitive.
In order to modify your commit, create a new one and squash it with the older one. Follow the guidelines from [`CONTRIBUTING.md`](https://github.com/mariasfiraiala/SO-exam/blob/master/CONTRIBUTING.md). Also, sign-off your commit and then we'll merge this PR.